File: app/src/main/java/com/example/myhomework/MainActivity.java
package com.example.myhomework;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.myhomework.R;

public class MainActivity extends AppCompatActivity {

    Button next1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        next1 = (Button)findViewById(R.id.btn_next1);
        next1.setOnClickListener(v -> {
            Intent intent = new Intent(this, com.example.myhomework.SecondActivity.class);
            startActivity(intent);
        });
    }

    public void onButtonClick(View v){

    }
}

File: app/src/main/java/com/example/mykotlinfirst/MainActivity.kt
package com.example.mykotlinfirst

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var vasya = CommonMotorist("Vasya", 150000, garage = ArrayList(10))
        var lada = Car("Lada Priora", 2012, 56000, vasya, 190000,
            'B', 7, 8,7, false)
        var petya = Reseller("Petr", 200000, garage = ArrayList(3))
        vasya.sellCar(lada, petya)
    }
}

inline fun printMessage(message: String){
    Log.e("lala", message)
}
